### DISCUSSION TITLE
How can Live demo result be different from local Docker

### DISCUSSION THREAD

**Discussion by ahjaworski** [gotenberg/gotenberg#369] [Q&A] [answered]
Hi, 
I don't know if this is the right place to ask this question, but I'm seeing different output when I run a request to the Live Demo compared to my own Docker image. Am I missing configuration that is enabled on the Live Demo or is there a different cause?

To reproduce:
_Live demo request:_
`curl --request POST 'https://live-demo-us.gotenberg.dev/forms/chromium/convert/url' --form 'url="https://jsfiddle.net/L8udx0c5/3/"' --form 'waitDelay="5s"' -o output.pdf`

_Local request:_
`docker run --rm -p 3000:3000 gotenberg/gotenberg:7`
`curl --request POST 'http://localhost:3000/forms/chromium/convert/url' --form 'url="https://jsfiddle.net/L8udx0c5/3/"' --form 'waitDelay="5s"' -o output2.pdf`

Now output.pdf contains a nice map at the bottom right, but output2.pdf does not.

-- update --
I'm running a Macbook M1. Ive asked a colleague with  Windows to perform the same steps and for him the local request is identical to the Live demo request. Another colleague with also a Macbook M1 has the same result Im seeing. So it seems the cause of the different results depends on the OS.

**Comment by gulien** [comment, marked as answer]
Hello @ahjaworski 👋

The live demo runs on an *amd64* platform, while your MacBook M1 runs on an *arm64* platform.

The main difference is that the *amd64* Docker image uses the latest Google Chrome version, while the *arm64* one relies on an older Chromium version.

See https://gotenberg.dev/docs/modules/chromium 😊

**Reply by ahjaworski** [reply]
Thanks for the quick and clear reply, I had not noticed that information in the documentation (which by the way is very nice to read and explanatory).
